docs/vectors/sealedbox_vectors.rst

```diff
@@ -0,0 +1,84 @@
+SealedBox reference vectors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Since libsodium's tests do not provide reference data for the SealedBox
+construction, the implementation is verified with a ``sealbox_test_vectors``
+utility program that produces and checks custom test vectors
+by making specific calls to libsodium API.
+
+To build the ``sealbox_test_vectors`` you need a ``C`` language compiler,
+a prebuilt libsodium library more recent than version 1.0.3 and the
+corresponding include headers.
+
+In a unix-like programming environment you shoud then execute:
+
+.. code-block:: bash
+
+    $ cc -o sealbox_test_vectors -l sodium sealbox_test_vectors.c
+
+The utility program can both generate list of vectors on standard
+output and check consistency of vectors pulled from standard input.
+
+Vector generation
+"""""""""""""""""
+
+When called with one or more command line arguments, ``sealbox_test_vectors``
+will generate the number of hex-encoded vectors requested as first argument,
+with the optional second and third arguments influencing the length of the
+generated messages:
+
+.. code-block:: bash
+
+    $ ./sealbox_test_vectors 1
+    XXXX...        XXXX...        XXXX...        XXXX...
```

Comment:
When I run `./sealbox_test_vectors 1` I often get three values instead of four as output. I'm wondering what's going on there.